src/libraries/System.Drawing.Common/src/System/Drawing/ImageAnimator.cs

```diff
@@ -381,10 +382,20 @@ public static void StopAnimate(Image image, EventHandler onFrameChangedHandler)
         private static void AnimateImages50ms()
         {
             Debug.Assert(s_imageInfoList != null, "Null images list");
+            DateTime animationStart = DateTime.Now;
```

Comment:
>DateTime [](http://example.com/codeflow?start=38&length=8)

did you consider using StopWatch instead?

Reply:
Thanks for the suggestion; I had not, but I think that could work nicely.